### PR TITLE
fix: top banner weird z-index position

### LINKED ISSUE #10080: [CAL-2151] Top banner has weird z-index position in settings
[https://github.com/calcom/cal.com/assets/8019099/10c87eb6-d4e3-4c2b-a66e-02adaa5945ad](https://github.com/calcom/cal.com/assets/8019099/10c87eb6-d4e3-4c2b-a66e-02adaa5945ad)

The banner should behave the same as on the regular pages.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2151](https://linear.app/calcom/issue/CAL-2151/top-banner-has-weird-z-index-position-in-settings)</sub>

### PR DESCRIPTION
**Pull request by puru-khedre**
## What does this PR do?

This PR fixes top banner's z-index position

Fixes #10080 

## Type of change

  - [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/features/settings/layouts/SettingsLayout.tsx [hunk 1]
```diff
@@ -215,7 +215,7 @@ const SettingsSidebarContainer = ({
   return (
     <nav
       className={classNames(
-        "no-scrollbar bg-muted fixed bottom-0 left-0 top-0 z-20 flex max-h-screen w-56 flex-col space-y-1 overflow-x-hidden overflow-y-scroll px-2 pb-3 transition-transform max-lg:z-10 lg:sticky lg:flex",
+        "no-scrollbar bg-muted fixed bottom-0 left-0 top-0 z-20 flex max-h-screen w-56 flex-col space-y-1 overflow-x-hidden overflow-y-scroll px-2 pb-3 transition-transform max-lg:z-10 lg:sticky lg:z-0 lg:flex",
```
Comment (Udit-takkar) [inline packages/features/settings/layouts/SettingsLayout.tsx:218]: @puru-khedre Although this works, shouldn't the top banner's z index be changed instead?

Comment (puru-khedre) [reply packages/features/settings/layouts/SettingsLayout.tsx:218]: I tried to fix it by changing the value of z-index but it didn't work.

Comment (N-NeelPatel) [reply packages/features/settings/layouts/SettingsLayout.tsx:218]: @puru-khedre - TopBanner component has `static` position, and z-index property doesn't work on `static` elements. Try changing it to `relative` and then add `z-index` property to see the changes.

Comment (PeerRich) [reply packages/features/settings/layouts/SettingsLayout.tsx:218]: @puru-khedre wanna try the above?

**Comment by Udit-takkar** [comment]
issue already fixed here https://github.com/calcom/cal.com/pull/10471

**Pull request closed without merging**